### ISSUE TITLE
Help modal: address "critical" and "major" accessibility issues

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/happy-heron#3193]
# Critical

- [x] Ensure form fields provide a programmatically associated label that describes the purpose of each field. 

From SODA: 
```Each of the form fields is duplicated in the Help modal dialog and the relationships between the for and id values are negated because they are duplicated.

Users may not be able to fill out form fields correctly, if there is no programmatically associated and visible label for each field.
```

# Major

- [x] Ensure each element has a unique ID

From SODA:

```Some elements are not well formed.  There are multiple id attributes with the same value that are referenced by other elements.  This occurs for each of the form fields that are duplicated in the Help dialog.

Though more of a usability issue, it is not clear why a user who might need help with form fields would gain assistance from a dialog with the same fields repeated.

When elements are not well formed, assistive technologies may not behave as intended.
```

- [x] Ensure each field has a meaningful example that communicates purpose.

From SODA:
```

There are common input fields that lack autocomplete attributes to communicate their purpose programmatically: the "what is your name" and "what is your email address" fields in the main screen and in the Help dialog.

When autocomplete is used to communicate the purpose of a field, users of assistive technology or users with cognitive disabilities can have access to icons or familiar terms indicating the field's purpose. In addition, when a form is filled out automatically using information the user has previously entered, the user does not need to decipher input fields that may appear differently in different forms and may not need to remember and re-enter data.
```
